package/MDAnalysis/analysis/encore/similarity.py

```diff
@@ -1283,7 +1273,7 @@ def dres(ensembles,
         classes from the dimensionality_reduction module. Different parameters
         for the same method can be explored by adding different instances of
         the same dimensionality reduction class. Provided methods are the
-        Stochastic Proximity Embedding (default) and the Principel Component
+        Stochastic Proximity Embedding (default) and the Principal Component
```

Comment:
This is a good question. The fact that there are now two versions of PCA in MDAnalysis is a bit of an accident. After early discussions on how encore could best be integrated in MDAnalysis, there was some consensus on making the dimensionality reduction and clustering code more generally available within MDAnalysis. I therefore rewrote the code so that both clustering and dimensionality got their own front-end functions, and that particular clustering and dimensionality reduction algorithms could just be plugged in - for instance those from scikit-learn. PCA was just a natural example to include. Only after doing this work did I realized that someone had coded a PCA in the mean time.

They should probably be merged at some point, but @orbeckst suggested that we stick with the two versions for this release, and then take a step back and think about some refactoring.

So, long story short. I haven't compared the two implementations. But the one in Encore is just a direct interface to the one in scikit-learn.